src/pages/grants/[id].tsx

```diff
@@ -130,7 +130,7 @@ const Grants = ({ slug }: GrantsDetailsProps) => {
                   <Flex
                     pos={{ base: 'fixed', md: 'static' }}
                     zIndex={999}
-                    bottom={0}
+                    bottom={12}
```

Comment:
this is a change from an earlier commit; should have checked out from staging again for the new PR

Reply:
hey abhishek actually no , The grants page is seperate , in the previous PR i just fixed the Grants and Projects page . as you can see i have pointed this in the issue aswell .

Reply:
ahh understood
my bad